app/views/manage/configs/index.html.haml

```diff
@@ -2,24 +2,53 @@
 
 .row
   .col-lg-6.mb-3
-    - @config.each_key do |key|
-      - value = @config[key]
-      .card.mb-3
-        .card-body
+    .card.mb-3
+      .card-body
+        %h4.card-title The Basics
```

Comment:
Thoughts on making these titles bolded? imo makes it a bit clearer
![Screen Shot 2020-12-01 at 5 14 01 PM](https://user-images.githubusercontent.com/2782749/100803132-9d5dac80-33f8-11eb-80fe-046c259d80b4.png)